test/CppTest/CppTest.vcxproj

```diff
@@ -324,10 +327,21 @@
       <Project>{b73ad907-6164-4294-88fb-f3c9c10da1f1}</Project>
       <Private>true</Private>
     </ProjectReference>
-    <ProjectReference Include="..\TestApps\AppLifecycleTestApp\AppLifecycleTestApp.vcxproj">	
-      <Project>{7c502995-59c3-483b-86ba-815985353633}</Project>	
+    <ProjectReference Include="..\TestApps\AppLifecycleTestApp\AppLifecycleTestApp.vcxproj">
```

Comment:
Why? I don't understand the reason for this change

Reply:
I don't know either.  I bet I mis-clicked and added the shared project by mistake.